<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>61</width>
      <height>601</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="freeHandDraw">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="drawLine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="drawRectangle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="drawCircle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pickTool">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="openButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="groupButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ungroupButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="imageButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>30</y>
      <width>81</width>
      <height>650</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="resizeButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Resize</string>
    </property>
   </widget>
   <widget class="QFrame" name="toolPanel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>321</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(217, 217, 217);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="toolCaneclButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>281</width>
       <height>131</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-28</y>
        <width>262</width>
        <height>157</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="plusButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="minusButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="multiplyButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="divideButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="equalButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="percentageButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="rootButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="cubeButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="integrationButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="piButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tool Box</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
